mrp_subcontracting_resupply_link/README.rst

```diff
@@ -0,0 +1,96 @@
+================================
+Mrp subcontracting resupply link
+================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmanufacture-lightgray.png?logo=github
+    :target: https://github.com/OCA/manufacture/tree/14.0/mrp_subcontracting_resupply_link
+    :alt: OCA/manufacture
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_subcontracting_resupply_link
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/129/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds the relationship between the purchase order and resupply and from resupply to the purchase order.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Go to *Manufacturing > Products > Products* and create a new subcontracted product with some Vendor (Purchase tab).
+#. Go to *Manufacturing > Products > Products* and create a new one with "Resupply Subcontractor on Order" route checked. [Component]
+#. Go to *Manufacturing > Products > Bill of materials* and create a new one with type = "Subcontracting" and set the subcontractor and component previously created.
+#. Go to *Purchase > Orders > Requests for Quotation* and create a new one to the subcontractor and add subcontracted product.
+#. Click on "Confirm order" button.
+#. A message linked to resupply(s) will have been added to purchase order.
```

Comment:
It doesn't. I guees this is the issue you were trying to figure out the other day, isn't it?

Reply:
This works correctly, what problem do you have with it?